logging/log_storage/log-storage-overview.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="log-storage-overview"]
+= About log storage
+:context: log-storage-overview
+
+toc::[]
+
+You can use a Loki or ElasticSearch log store on your cluster for storing logs, or you can use a xref:../../logging/log_collection_forwarding/log-forwarding.adoc#logging-create-clf_log-forwarding[`ClusterLogForwarder` custom resource (CR)] to forward logs to an external store. Several types of store are supported, including fluentd, rsyslog, kafka and others.
```

Comment:
> Several types of store are supported, including fluentd, rsyslog, kafka and others.

Do we need to make this clearer that these are the options for the forwarding? "Log Storage" itself only supports ElasticSearch (the older 6.8 version) and Loki (using the Loki Operator, so branded as "LokiStack")

Reply:
Opting to remove this sentence entirely. We document all the supported outputs in the collection/forwarding docs section, so IMO there's no need to repeat this here. I am currently working on https://issues.redhat.com/browse/OBSDOCS-134 also to update the section about forwarding logs to internal stores, so I can maybe add a link to and/or include those docs in the log storage section to help complete the user story here.